test/OrchardCore.Tests/Roles/Extensions/SystemRoleProviderExtensionsTests.cs

```diff
@@ -1,32 +0,0 @@
-using OrchardCore.Environment.Shell;
-
-namespace OrchardCore.Roles.Tests;
-
-public class SystemRoleProviderExtensionsTests
-{
-    [Theory]
-    [InlineData("Administrator", true)]
-    [InlineData("ADMINISTRATOR", true)]
-    [InlineData("administrator", true)]
-    [InlineData("AdminiSTratoR", true)]
-    [InlineData("Test", false)]
-    [InlineData("TEST", false)]
-    [InlineData("TesT", false)]
-    [InlineData("test", false)]
-    public void IsAdminRole_ReturnsTrue_IfTheRoleExists(string roleName, bool expectedResult)
```

Comment:
Why was this deleted or moved?

Reply:
moved

Reply:
I think we should respect the classes and namespaces, Please revert this change, and let us merge this. We already align the tests to actual namespaces in the source

Reply:
All these tests are testing the same thing. There is no need for having a second class.